README.md

```diff
@@ -78,6 +78,22 @@ $ ./awe -p /path/to/the/alan/wake/data/directory
 ```
 
 
+Controls
+--------
+OpenAWE features a free camera that can be controlled with keyboard and mouse. Current mappings stand as follows:
+```markdown
+# Movement
+- [WASD] to move forwards/backwards/left/right
+- [RF] to move up/down
+- [TG] to increase/decrease movement speed
+# Camera
+- [←→↑↓] to look left/right/up/down
+- [YH] to increase/decrease camera speed
```

Comment:
I assume you use a QWERTY keyboard? Since I am using a german QWERTZ keyboard this combination is a little bit counter intuitive for me. Maybe you have an idea how to handle this better, if not we can leave it this way for now.

Reply:
Yes, when I'm referring to keys here, I imply QWERTY layout. Also not sure yet of what can be done here for now.